src/room/CallEndedView.tsx

```diff
@@ -51,7 +53,7 @@ export function CallEndedView({ client }) {
               variant="default"
               to="/register"
             >
-              Create account
+              <>Create account</>
```

Comment:
what was this for?

Reply:
children was of type JSX.Element. I changed it to `JSX.Element | string`